### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by welfare520**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -93,12 +93,12 @@ Your application will be reviewed by at least two of our engineers. We do take i
...
-- **Code quality**: is the code simple, easy to understand, and maintainable? Are there any code smells or other red flags? Does object-oriented code follows principles such as the single responsibility principle? Is the coding style consistent with the language's guidelines? Is it consistent throughout the codebase?
-- **Testing**: how thorough are the automated tests? Will they be difficult to change if 
-the requirements of the application were to change? Are there some unit and some integration tests? We're not looking for full coverage (given time constraint) but just trying to get a feel for your testing skills.
+- **Correctness**: does the application do what was asked? If anything is missing, does the README explain why it is missing?
+- **Code quality**: is the code simple, easy to understand, and maintainable? Is there any code smells or other red flags? Does the object-oriented code follow principles such as the single responsibility principle? Is the coding style consistent with the language's guidelines? Is it consistent throughout the codebase?
```
Comment (levrik) [inline readme.md]: Actually `Are` was correct here.

File: readme.md [hunk 2]
```diff
@@ -93,12 +93,12 @@ Your application will be reviewed by at least two of our engineers. We do take i
...
-- **Testing**: how thorough are the automated tests? Will they be difficult to change if

-the requirements of the application were to change? Are there some unit and some integration tests? We're not looking for full coverage (given time constraint) but just trying to get a feel for your testing skills.
+- **Correctness**: does the application do what was asked? If anything is missing, does the README explain why it is missing?
+- **Code quality**: is the code simple, easy to understand, and maintainable? Is there any code smells or other red flags? Does the object-oriented code follow principles such as the single responsibility principle? Is the coding style consistent with the language's guidelines? Is it consistent throughout the codebase?
+- **Testing**: how thorough are the automated tests? Will it be difficult to change if

+the application required were to change? Are there some units and some integration tests? We're not looking for full coverage (given time constraint) but just trying to get a feel for your testing skills.
```
Comment (levrik) [inline readme.md]: Actually the `Testing` section was also correct already. Now the meaning was changed.

File: readme.md (hunk 1 above)
Comment (levrik) [reply readme.md]:
```suggestion
- **Code quality**: is the code simple, easy to understand, and maintainable? Are there any code smells or other red flags? Does the object-oriented code follow principles such as the single responsibility principle? Is the coding style consistent with the language's guidelines? Is it consistent throughout the codebase?
```

File: readme.md (hunk 2 above)
Comment (welfare520) [reply readme.md]: right

File: readme.md (hunk 1 above)
Comment (mize85) [inline readme.md]: _Is there any code smells_
well, **Are** is corrrect here, smells/flags is plural 🤔

File: readme.md (hunk 2 above)
Comment (mize85) [inline readme.md]: **they** was refering to the tests, **it** does not fit here
_if  the requirements of the application were to change?_ was ok too :)
_units and some integration tests?_ should stay singular **unit**

**Pull request merged**